docs/Makefile

```diff
@@ -29,7 +29,7 @@ SVG2PDF       = inkscape
 SVG2PDF_FLAGS =
 
 # Internal variables.
-ALLSPHINXOPTS   = -t oss -d $(BUILDDIR)/doctrees $(SPHINXOPTS) .
+ALLSPHINXOPTS   = -t iso -d $(BUILDDIR)/doctrees $(SPHINXOPTS) .
```

Comment:
You probably didn't mean to commit this?

Reply:
thx